packages/widgets/src/docklayout.ts

```diff
@@ -1181,7 +1190,7 @@ export namespace DockLayout {
      *
      * @returns A new tab bar for a dock layout.
      */
-    createTabBar(): TabBar<Widget>;
+    createTabBar(document: Document | ShadowRoot): TabBar<Widget>;
```

Comment:
I think this argument should be optional

Reply:
Change made as suggested  (FWIW it only actually gets called from one place, within the DockPanel),